apps/platform/trpc/ratelimit.ts

```diff
@@ -0,0 +1,73 @@
+import {
+  NoopRatelimit,
+  Ratelimit,
+  type RatelimitConfig
+} from '@unkey/ratelimit';
+import { trpcContext } from './trpc';
+import { env } from '~platform/env';
+import { TRPCError } from '@trpc/server';
+import type { TrpcContext } from '~platform/ctx';
+
+export const ipIdentifier = (ctx: TrpcContext) =>
+  `ip:${ctx.event.env.incoming.socket.remoteAddress ?? 'unknown'}`;
+export const accountIdentifier = (ctx: TrpcContext) =>
+  `account:${ctx.account?.id!}`;
+export const orgIdentifier = (ctx: TrpcContext) => `org:${ctx.org?.id!}`;
+
+const CACHE_MAP = new Map<string, Ratelimit | NoopRatelimit>();
+
+type CreateRateLimiterConfig = {
+  /**
+   * Number of requests allowed
+   */
+  limit: number;
+  /**
+   * Duration of the rate limit window
+   * @default '1h'
+   */
+  duration?: RatelimitConfig['duration'];
+  /**
+   * Namespace for the rate limit
+   * @default 'public'
+   */
+  namespace?: RatelimitConfig['namespace'];
+  /**
+   * Function to create the identifier for the rate limit
+   * @default ipIdentifier
+   */
+  createIdentifier?: (ctx: TrpcContext) => string;
+};
+
+export const createRatelimit = ({
```

Comment:
rename to ratelimiter to make it clearer/cleaner

so in usage its `.use(ratelimiter({ limit: 30 })